dialogue.py

```diff
@@ -4,9 +4,12 @@
 # Invokes run.sh with configurations stored in the file
 # 'runtype_options.json'
 
-
 import json
 
+EXIT_CONSTS = set(['q', 'quit', 'exit'])
+YES_CONSTS = set(['y', 'yes'])
```

Comment:
Interesting use of set, haven't seen this pattern before. Works well enough for what we need it to do, I'm guessing.

Reply:
Any other solution would just improve code clarity rather than function, so I'm more for letting this in and changing it later if we need.